packages/aws-cdk-lib/aws-rds/README.md

```diff
@@ -854,7 +854,32 @@ const instance = new rds.DatabaseInstance(this, 'Instance', {
 });
 ```
 
-**Note**: In addition to the setup above, you need to make sure that the database instance has network connectivity
+The following example shows enabling domain support for a Aurora databse cluster and creating an IAM role to access Directory Services.
+
+```ts
+declare const vpc: ec2.Vpc;
+const iamRole = new iam.Role(this, 'Role', {
+  assumedBy: new iam.CompositePrincipal(
+    new iam.ServicePrincipal('rds.amazonaws.com'),
+    new iam.ServicePrincipal('directoryservice.rds.amazonaws.com'),
+  ),
+  managedPolicies: [
+    iam.ManagedPolicy.fromManagedPolicyArn(this, 'RdsRole', 'arn:aws:iam::aws:policy/service-role/AmazonRDSDirectoryServiceAccess'),
+  ],
+});
+
+new rds.DatabaseCluster(this, 'Database', {
+  engine: rds.DatabaseClusterEngine.auroraMysql({ version: rds.AuroraMysqlEngineVersion.VER_3_05_1 }),
+  writer: rds.ClusterInstance.provisioned('Instance', {
+    instanceType: ec2.InstanceType.of(ec2.InstanceClass.BURSTABLE3, ec2.InstanceSize.MEDIUM),
+  }),
+  vpc,
+  domain: 'd-????????', // The ID of the domain for the instance to join.
```

Comment:
I'm not familiar with Kerberos Authentication so I'm not sure, but would cluster be more appropriate than instance?
```suggestion
  domain: 'd-????????', // The ID of the domain for the cluster to join.
```